### PR TITLE
pkcon: add page

### PR DESCRIPTION
**Pull request by mikerossiter**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by mikerossiter** [comment]
I don't understand why this is failing. I can't see any whitespace to delete.

File: pages/linux/pkcon.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- Install a package:
+
+`pkcon install {{package}}`
+
+- Remove a package:
+
+`pkcon remove {{package}}`
+
+- Refresh the package cache (similar to `sudo apt update`):
```
Comment (CleanMachine1) [inline pages/linux/pkcon.md:15]:
```suggestion
- Refresh the package cache:
```

File: pages/linux/pkcon.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# pkcon
+
+> Command line client for PackageKit console program used by Discover and Gnome software and alternative to 'apt'.
+> Mostly used by Plasma based desktop environments such as KDE Neon and Kubuntu although it is distro-agnostic.
```
Comment (CleanMachine1) [inline pages/linux/pkcon.md:4]:
```suggestion
```

File: pages/linux/pkcon.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+> More information: <https://manpages.ubuntu.com/manpages/bionic/man1/pkcon.1.html>.
```
Comment (CleanMachine1) [inline pages/linux/pkcon.md:5]:
```suggestion
> More information: <https://manned.org/pkcon>.
```

**Comment by mikerossiter** [comment]
Accepted commit suggestions

**Review by CleanMachine1 — APPROVED** [review]
lgtm, thanks for the contribution.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. And welcome to tldr btw.

**Pull request merged**